app/models/template_kind.rb

```diff
@@ -27,4 +27,8 @@ def self.add_to_jar(hash)
   def to_s
     TemplateKind.jar[name] || name
   end
+
```

Comment:
This is kind of unnecessary, as it forces changing the view too. Can you change `to_s` to do this?

```ruby
_(TemplateKind.jar[name] || name)
```